templates/wordpress/files/README.md

```diff
@@ -42,3 +33,13 @@ define('LOGGED_IN_KEY', 'LOGGED_IN_SALT');
 define('NONCE_KEY', 'NONCE_SALT');
 define('AUTH_SALT', 'SECURE_AUTH_SALT');
 ```
+
+## Customizations
+
+The following changes have been made relative to Drupal 8 as it is downloaded from Drupal.org.  If using this project as a reference for your own existing project, replicate the changes below to your project.
```

Comment:
Drupal 8?

Reply:
There are a *lot* of changes from Drupal 8 to WordPress...